.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": ""
+}
```

Comment:
probably worth not committing the `.vscode` (you can add it to the .gitignore file) and then untrack it

Reply:
can you try to remove this file.